Specs/DataSources/PolygonGraphicsSpec.js

```diff
@@ -291,6 +303,7 @@ describe("DataSources/PolygonGraphics", function () {
     );
     testDefinitionChanged(property, "arcType", ArcType.GEODESIC, ArcType.RHUMB);
     testDefinitionChanged(property, "zIndex", 54, 3);
+    testDefinitionChanged(property, "textureCoordinates", [], []);
```

Comment:
Would be better to use two different values in this test.

Reply:
Updated the test case to use different values for `textureCoordinates` and `hierarchy`.